pyramid_oereb/views/webservice.py

```diff
@@ -81,6 +88,7 @@ def get_versions(self):
         response = render_to_response(renderer_name, versions, request=self._request)
         if self._is_json():
             response.content_type = 'application/json; charset=UTF-8'
+        response.extras = OerebStats(service='GetVersions', output_format=renderer_name)
```

Comment:
There should be a catch-all type of logic around this, with a WARN log entry, so that a mistake in Stats logic can never cause a request to fail.
Apart from that, it is looking good to me, and I am looking forward to the next features.